src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
```

Comment:
Pop() method modifies the original array, this can cause an unexpected behavior, I would replace it with access by last index